segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -65,6 +66,7 @@
             .garbageCollectionMaxQueueSize(16 * 1024)
             .garbageCollectionSleep(Duration.ofSeconds(60))
             .garbageCollectionMaxAttempts(3)
+            .indexBlockSize(64 * 1024 * 1024)
```

Comment:
Why don't you use the default values from the properties you defined above? Let's change this for all of these.

Reply:
I'll fix all those in https://github.com/pravega/pravega/pull/5539